docs/reports/requirements/5_tech-stack.md

```diff
@@ -1,19 +1,80 @@
-# Tech Stack
+# SLIDE CONTENT
+## Tech Stack
 
 Due to the various approaches that could be taken for this project, our client has left us with the choice of what tech 
 stack to use. Through our research, two options presented themselves: Node.js + Typescript & Python + Django. Because of
 the familiarity our group already has with languages used in the existing tech stack we have chosen to use Node + 
 Typescript for our backend implementation.
 
-## Front-End
+### Front-End
 HTML & CSS
 In order to maintain simplicity, any front-end development which may be required should be done in simple HTML/CSS
 Styling must stay consistent with Kukarella’s - if additional frameworks such as React are required, they may be added.
 Kukarella does not currently request a front-end, though did mention it may be an additional feature if time permits.
-## Back-End
+### Back-End
 Node.js & Typescript
 Our backend code will be built using Typescript - this provides us with strict control over types, and aligns with the 
 already existing code within the Kukarella system. We will be using Node.js in order to allow us to build our service as
 an easily maintainable API. The library “Rhubarb Lip Sync” is currently required for our backend implementation. This is
 a command line tool controllable by various programming languages. Its last update was June 14, 2022
 
+# PDF CONTENT
+# Tech Stack
+
+There are numerous approaches which could be taken when determining what the best course of action for this
+project could be. Due to this, Kukarella has left our choice of tech stack open. The only requirement they have is that
+it must either integrate easily into their existing tech stack or it be an API which they may call and receive a
+generated video from. Research done as a group has provided us with three separate tech stacks which could be used. The
+tech stacks identified are outlined in the table below, and discussion and rational for our chosen tech stack (bolded)
+will follow.
```

Comment:
```suggestion
```

Reply:
I think this plays into that last suggestion as well, would be weirder without saying will follow